integration_tests/e2e/overview/preInductionOverview.cy.ts

```diff
@@ -100,16 +77,12 @@ context('Prisoner Overview page - Pre Induction', () => {
     overviewPage.clickMakeProgressPlan()
 
     // Then
-    Page.verifyOnPage(CreateCiagInductionPage)
+    Page.verifyOnPage(HopingToWorkOnReleasePage) //
```

Comment:
This is the crux of the change in this cypress test in respect of the new page implementation